massgov/mayflower
# Feature/dp 12216 banner images

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description
Update banner image scss to avoid background repeating.

## Related Issue / Ticket

https://jira.mass.gov/browse/DP-12216


## Steps to Test

1. Go Here: locations/barnstable-probate-and-family-court 
2. You Should See This: The banner image next to the map should not repeat at larger width browsers

## Screenshots
N/A


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
